packages/react-query/src/FormatBalance.tsx

```diff
@@ -84,17 +86,17 @@ function FormatBalance ({ children, className = '', format, formatIndex, isShort
   // labelPost here looks messy, however we ensure we have one less text node
   return (
     <div className={`ui--FormatBalance ${className}`}>
-      {label ? <>{label}&nbsp;</> : ''}
-      <span
-        className='ui--FormatBalance-value'
-        data-testid='balance-summary'>{
+      {label ? <>{label}&nbsp;</> : ''}<span className='ui--FormatBalance-value'
+        data-testid='balance-summary' >{
           valueFormatted
             ? splitFormat(valueFormatted, labelPost, isShort)
             : value
               ? value === 'all'
                 ? t<string>('everything{{labelPost}}', { replace: { labelPost } })
                 : applyFormat(value, formatInfo, withCurrency, withSi, isShort, labelPost)
-              : `-${labelPost || ''}`
+              : labelPost instanceof String
```

Comment:
```suggestion
              : isString(labelPost)
```

Which may pickup `String` as well (so `.toString()` could be required below)

Reply:
/cc @vitaliyworks :)

Reply:
I still think that `typeof(labelPost) === 'string'` should be used because `labelPost` cannot be of type `String`

Reply:
Since you pass in a string, the isString() should do the right thing. Plus, it aligns with the idioms used in polkadot-js.

Reply:
✅